### PR TITLE
Do not print verbose info about app updates if there are none

### PR DESCRIPTION
**Pull request by ChristophWurst**
This is cosmetic but if you have a large number of apps installed then
you'll see a wall of text during the server and app upgrade when it
tries to update each app via the app store. In may cases nothing will be
updated. For those boring cases we can hide the verbose info, but show
when occ is run with -v. Any actual update will still print a few lines.
Those are the important ones for the admin.

## Before

```
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Setting log level to debug
Turned on maintenance mode
Updating database schema
Updated database
Disabled incompatible app: mail
Updating <calendar> ...
Updated <calendar> to 2.4.0-RC.2
Checking for update of app accessibility in appstore
Checked for update of app "accessibility" in App Store 
Checking for update of app admin_audit in appstore
Checked for update of app "admin_audit" in App Store 
Checking for update of app calendar in appstore
Checked for update of app "calendar" in App Store 
Checking for update of app cloud_federation_api in appstore
Checked for update of app "cloud_federation_api" in App Store 
Checking for update of app comments in appstore
Checked for update of app "comments" in App Store 
Checking for update of app contactsinteraction in appstore
Checked for update of app "contactsinteraction" in App Store 
Checking for update of app dashboard in appstore
Checked for update of app "dashboard" in App Store 
Checking for update of app dav in appstore
Checked for update of app "dav" in App Store 
Checking for update of app federatedfilesharing in appstore
Checked for update of app "federatedfilesharing" in App Store 
Checking for update of app federation in appstore
Checked for update of app "federation" in App Store 
Checking for update of app files in appstore
Checked for update of app "files" in App Store 
Checking for update of app files_external in appstore
Checked for update of app "files_external" in App Store 
Checking for update of app files_sharing in appstore
Checked for update of app "files_sharing" in App Store 
Checking for update of app files_trashbin in appstore
Checked for update of app "files_trashbin" in App Store 
Checking for update of app files_versions in appstore
Checked for update of app "files_versions" in App Store 
Checking for update of app lookup_server_connector in appstore
Checked for update of app "lookup_server_connector" in App Store 
Checking for update of app oauth2 in appstore
Checked for update of app "oauth2" in App Store 
Checking for update of app provisioning_api in appstore
Checked for update of app "provisioning_api" in App Store 
Checking for update of app settings in appstore
Checked for update of app "settings" in App Store 
Checking for update of app sharebymail in appstore
Checked for update of app "sharebymail" in App Store 
Checking for update of app systemtags in appstore
Checked for update of app "systemtags" in App Store 
Checking for update of app theming in appstore
Checked for update of app "theming" in App Store 
Checking for update of app twofactor_backupcodes in appstore
Checked for update of app "twofactor_backupcodes" in App Store 
Checking for update of app updatenotification in appstore
Checked for update of app "updatenotification" in App Store 
Checking for update of app user_ldap in appstore
Checked for update of app "user_ldap" in App Store 
Checking for update of app user_status in appstore
Checked for update of app "user_status" in App Store 
Checking for update of app viewer in appstore
Checked for update of app "viewer" in App Store 
Checking for update of app weather_status in appstore
Checked for update of app "weather_status" in App Store 
Checking for update of app workflowengine in appstore
Checked for update of app "workflowengine" in App Store 
Checking for update of app mail in appstore
Checked for update of app "mail" in App Store 
Update successful
Turned off maintenance mode
Resetting log level
```

## After

```
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Setting log level to debug
Turned on maintenance mode
Updating database schema
Updated database
Updating <calendar> ...
Updated <calendar> to 2.4.0-RC.3
Update successful
Turned off maintenance mode
Resetting log level
```

### CODE REVIEW

File: core/Command/Upgrade.php [hunk 1]
```diff
@@ -205,13 +205,17 @@ function ($success) use ($output, $self) {
 				$output->writeln('<comment>Disabled incompatible app: ' . $app . '</comment>');
 			});
 			$updater->listen('\OC\Updater', 'checkAppStoreAppBefore', function ($app) use ($output) {
-				$output->writeln('<info>Checking for update of app ' . $app . ' in appstore</info>');
+				if ($output->isVerbose()) {
```
Comment (nickvergessen) [inline core/Command/Upgrade.php:208]: I use -v to show the timestamps
I really think those messages are not adding any benefit and should be removed

Comment (ChristophWurst) [reply core/Command/Upgrade.php:208]: Also fine by me :)

**Review by nickvergessen — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1: please make fixup bot happy

**Review by CarlSchwan — APPROVED** [review]

**Review by MichaIng — APPROVED** [review]

**Comment by MichaIng** [comment]
Great, I was annoyed but this as well. I vote for backport to Nextcloud 23 at least.

**Pull request merged**